wdae/tests/integration/test_wdae_config/conftest.py

```diff
@@ -0,0 +1,155 @@
+# pylint: disable=W0621,C0114,C0116,W0212,W0613,C0415,
+# type: ignore
+
+import os
+import sys
+import contextlib
+
+import pytest
+
+
+class LiveServer:
+    """The liveserver fixture.
+
+    Copied from `pytest_django`: https://github.com/pytest-dev/pytest-django/
```

Comment:
Can you also describe what is the difference between this class and Django's class and why you've copied it.

Reply:
This class is an exact copy of the one from the `pytest-django` plugin. The problem with using this class directly from the `pytest-django` package is that when I import the `pytest_django.liver_server_helpers.LiveServer` class, it triggers the initialization of the `pytest_django` plugin, which initializes our Django application. And there is no way (or at least I was unable to find one) to control how the Django application is initialized.

Reply:
OK. Just copy paste this explanation in the docstring for the class.

Reply:
Done.